rust/migrations/202211170900_create_cfd_table.sql

```diff
@@ -5,15 +5,15 @@ CREATE TABLE IF NOT EXISTS cfd_state (
 INSERT INTO
     cfd_state (id, state)
 VALUES
```

Comment:
If you change the sql file you likely will need to run the migration again or they get out of sync.